pkg/reconciler/buildrun/resources/sources/bundle.go

```diff
@@ -21,6 +22,12 @@ func AppendBundleStep(
 	source build.Source,
 	name string,
 ) {
+	// append the result
+	taskSpec.Results = append(taskSpec.Results, pipeline.TaskResult{
+		Name:        fmt.Sprintf("%s-source-%s-bundle-image-digest", prefixParamsResultsVolumes, name),
```

Comment:
```suggestion
		Name:        fmt.Sprintf("%s-source-%s-image-digest", prefixParamsResultsVolumes, name),
```

To save some bytes. Needs changes at other places.